ui/src/pages/FlowRuns.vue

```diff
@@ -108,6 +108,9 @@
   bg-opacity-90
   py-2
   z-10
+  bg-background
```

Comment:
By chance did you test this in dark mode? I think you might need a `dark:bg-background-400` or similar

Reply:
is there even dark mode in OSS?..

Reply:
There is. 

I thought the point of the background and foreground classes was using one class would give you the right value for both light and dark mode?

Reply:
This is a good suggestion, adjusting shade does make this top bar look better